src/main/resources/application.conf

```diff
@@ -43,10 +43,18 @@ datastax-java-driver {
       show-stack-traces = true
     }
   }
-  basic.request.timeout = 10 seconds
+  basic.request.timeout = 20 seconds
   profiles {
     slow {
       basic.request.timeout = 30 seconds
     }
+
+    ddl {
+      basic.request.timeout = 30 seconds
+    }
+
+    count {
+      basic.request.timeout = 60 seconds
```

Comment:
This sounds bit high (as does basic request timeout of 20 secs), but easy enough to change later on I suppose.